packages/stream_video/lib/src/stream_video.dart

```diff
@@ -84,39 +129,16 @@ class StreamVideo {
 
   static final InstanceHolder _instanceHolder = InstanceHolder();
 
-  static StreamVideo init(
-    String apiKey, {
-    String coordinatorRpcUrl = _defaultCoordinatorRpcUrl,
-    String coordinatorWsUrl = _defaultCoordinatorWsUrl,
-    LatencySettings latencySettings = const LatencySettings(),
-    RetryPolicy retryPolicy = const RetryPolicy(),
-    SdpPolicy sdpPolicy = _defaultSdpPolicy,
-    Priority logPriority = Priority.none,
-    LogHandlerFunction logHandlerFunction = _defaultLogHandler,
-    bool muteVideoWhenInBackground = false,
-    bool muteAudioWhenInBackground = false,
-  }) {
-    _setupLogger(logPriority, logHandlerFunction);
-    return _instanceHolder.init(
-      apiKey,
-      latencySettings: latencySettings,
-      retryPolicy: retryPolicy,
-      sdpPolicy: sdpPolicy,
-      muteVideoWhenInBackground: muteVideoWhenInBackground,
-      muteAudioWhenInBackground: muteAudioWhenInBackground,
-    );
-  }
-
   /// The singleton instance of the Stream Video client.
   static StreamVideo get instance => _instanceHolder.instance;
 
   /// Resets the singleton instance of the Stream Video client.
   ///
   /// This is useful if you want to re-initialise the SDK with a different
   /// API key.
-  static Future<void> reset({bool disconnectUser = false}) async {
-    if (disconnectUser) {
-      await _instanceHolder.instance.disconnectUser();
+  static Future<void> reset({bool disconnect = false}) async {
+    if (disconnect) {
+      await _instanceHolder.instance.disconnect();
     }
     return _instanceHolder.reset();
   }
```

Comment:
`_instanceHolder` won't always contain the instance as we now have it configurable.
```dart
options.installAsSingleton
```